src/InvokePlaster.ps1

```diff
@@ -1133,6 +1133,8 @@ function Invoke-Plaster {
 
                         # Eval script expression delimiters
                         if ($content -and ($content.Count -gt 0)) {
+                            # Sets the Variable to the currently used template
+                            Set-Variable -Name PLASTER_TemplateFileSource -Value $srcPath -Scope Script
```

Comment:
Unfortunately this doesn't work "as-is" because setting a variable in this runspace doesn't help since the template file is processed in a constrained runspace.  The variable needs to be set there.  I'll noodle on this some as I really like the idea of providing this info inside a template file.